<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>projectdialog</class>
 <widget class="QDialog" name="projectdialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>471</width>
    <height>337</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Code:</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Manager:</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Name:</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Budget:</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLineEdit" name="lineEditCode">
           <property name="styleSheet">
            <string notr="true">background-color:rgb(238, 238, 236);
color:rgb(0, 0, 170);</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditManager">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(136, 138, 133);</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditName">
           <property name="styleSheet">
            <string notr="true">background-color:rgb(238, 238, 236);
color:rgb(0, 0, 170);</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditBudget">
           <property name="styleSheet">
            <string notr="true">background-color:rgb(238, 238, 236);
color:rgb(0, 0, 170);</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBoxActive">
       <property name="text">
        <string>Active</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QPushButton" name="pushButtonSubmit">
         <property name="text">
          <string>Submit</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonCancel">
         <property name="text">
          <string>Cancel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonSubmit</sender>
   <signal>clicked()</signal>
   <receiver>projectdialog</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>126</x>
     <y>264</y>
    </hint>
    <hint type="destinationlabel">
     <x>119</x>
     <y>297</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
